src/components/EditableTagField/__snapshots__/EditableTagField.stories.storyshot

```diff
@@ -111,7 +111,7 @@ exports[`storybook Storyshots Components/Editable Tags Field Editable Tag Field
         <button
```

Comment:
Strangely the grouped badges are messed up for me when I look at a version, but not on the My Patches page

<img width="319" alt="Screen Shot 2022-11-08 at 2 15 50 PM" src="https://user-images.githubusercontent.com/47064971/200656029-4e41651a-4aa5-4019-9121-2b0db5197dd1.png">